### PR TITLE
Add Flagsmith to Miscellaneous section

### PR DESCRIPTION
**Pull request by matthewelwell**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding has working installation instructions.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

It would be great to add a new section for Feature Management tools as there are a number of open source options out there (e.g. Go Feature Flag, flagd) but I couldn't add a new section _and_ 3 items based on the rules set out above.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1328,6 +1328,7 @@ Software that does not fit in another section.
 - [DomainMOD](https://domainmod.org) - Application to manage your domains and other internet assets in a central location. DomainMOD includes a Data Warehouse framework that allows you to import your WHM/cPanel web server data so that you can view, export, and report on your data. ([Demo](https://demo.domainmod.org), [Source Code](https://github.com/domainmod/domainmod)) `GPL-3.0` `PHP`
 - [DOMJudge](https://www.domjudge.org/) - A system for running a programming contest, like the ICPC regional and world championship programming contests. ([Demo](https://www.domjudge.org/demo), [Source Code](https://github.com/DOMjudge/domjudge)) `GPL-2.0/BSD-3-Clause/MIT` `PHP`
 - [F-Droid](https://f-droid.org) - Server tools for maintaining an F-Droid repository system. ([Source Code](https://gitlab.com/fdroid/fdroidserver)) `AGPL-3.0` `Python/Docker`
+- [Flagsmith](www.flagsmith.com) - We help teams ship faster and continuously improve digital products with feature flags. Alternative to LaunchDarkly. ([Source Code](https://github.com/flagsmith/flagsmith)) `BSD3` `python`
```
Comment (Rabenherz112) [inline README.md:1331]:
```suggestion
- [Flagsmith](www.flagsmith.com) - We help teams ship faster and continuously improve digital products with feature flags. Alternative to LaunchDarkly. ([Source Code](https://github.com/flagsmith/flagsmith)) `BSD-3-Clause` `Docker`
```

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]
Hey there, As you exclusively offer installation options through Docker or Kubernetes, I have modified the language tag accordingly.

Furthermore, I have a question regarding the presence of a secondary license mentioned in [api/permissions/LICENSE.md](https://github.com/Flagsmith/flagsmith/blob/main/api/permissions/LICENSE.md). Could you clarify if this license is a custom one? What specific purpose does it serve? From my interpretation, it appears to ensure that users purchase additional licenses when they require additional features. Is my understanding correct?

**Review by nodiscc — CHANGES_REQUESTED** [review]
The description should explain what the software does.

File: README.md (hunk 1 above)
Comment (matthewelwell) [reply README.md:1331]: >Furthermore, I have a question regarding the presence of a secondary license mentioned in [api/permissions/LICENSE.md](https://github.com/Flagsmith/flagsmith/blob/main/api/permissions/LICENSE.md). Could you clarify if this license is a custom one? What specific purpose does it serve? From my interpretation, it appears to ensure that users purchase additional licenses when they require additional features. Is my understanding correct?

Yes, this license has been customised. It prevents unauthorised used of the role based access functionality, for which some of the source code resides in the open source repository. This does not prevent users from deploying or using the software, it simply limits this one specific area of functionality.

Comment (matthewelwell) [reply README.md]: I have applied the suggested changes.

**Comment by matthewelwell** [comment]
> The description should explain what the software does.

I have updated the description. Let me know if it's still not clear.

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**